print-your-cert-controller

```diff
@@ -94,29 +98,32 @@ kubectl get cert -ojson --watch | jq -c --unbuffered | while read -r cert; do
         out=
         brother_rc=
 
-        if [[ -n $mock ]]; then
-            out="mocked output for front print"
-            brother_rc=0
-        else
-            out=out=$(brother_ql --model QL-820NWB --printer usb://0x04f9:0x209d print --label 62 back-$certname.png 2>&1)
-            brother_rc=$?
-        fi
-
-        if [[ $brother_rc != 0 ]]; then
-            printf "%s: failed to print back-$certname.png: %s\n" "$(jq -r .metadata.name <<<"$cert")" "$(tail -1 <<<"$out")"
-            if ! out=$(kubectl patch cert "$(jq -r .metadata.name <<<"$cert")" --subresource status --type=json -p '
-            [{
-                "op": "add", "path": "/status/conditions", "value":[{
-                    "type": "Printed",
-                    "status": "False",
-                    "reason": "Error",
-                    "message": "brother_ql: '"$(tail -1 <<<"$out")"'"
-                }]
-            }]' 2>&1); then
-                printf "%s: failed to set the 'Printed' condition to 'False': %s\n" "$(jq -r .metadata.name <<<"$cert")" "$(tr $'\n' ' ' <<<"$out")"
-            fi
-            continue
-        fi
+        # This is commented as in SLC we printed labels on one big sticker rather than two smaller stickers
+        # But we might need to uncomment to return to using smaller stickers
```

Comment:
We ran out of larger stickers now too, maybe it's a good idea to add a commit that reverts to the smaller stickers so we are ready for KubeCon London (buying smaller stickers in europe is quite easy).

Reply:
I think I'd probably vote to keep this as-is for now just to minimise the amount of work on this. This is pretty trivial to revert for London!